test/unit-tests/org/orgTest.js

```diff
@@ -570,163 +570,10 @@ describe('Test Org Controller', () => {
         })
     })
 
-    it('Org role does not exist (Adding multiple roles)', (done) => {
-      // perform the request to the api
-      chai.request(server)
-        .post('/api/test/cna/' + existentOrg.short_name + '?active_roles.add="ADP"&active_roles.add="vendor"')
```

Comment:
Just as a note, we have to make sure to keep a list of these test cases so we can move them to the integration suite

Reply:
Agree